test/epanet_test_db_unversioned.sql

```diff
@@ -1,40 +0,0 @@
-CREATE EXTENSION IF NOT EXISTS postgis;
```

Comment:
Isn't there a test where we test the versioning from this file?

Reply:
Has it been modified or integrated elsewhere?

Reply:
I change the way we load data in test, data is always unversioned then we version it with versioningDB. Anyway I didn't see this tests was failing because of its name (not ending with _test.py). This is fixed now 3f82c2c